Formula/zrepl.rb

```diff
@@ -28,13 +28,13 @@ def install
 
     ENV["GOPATH"] = gopath
     ENV.prepend_create_path "PATH", gopath/"bin"
+    ENV["GO111MODULE"] = "on"
```

Comment:
we recently removed this option in lots of golang formula. So I think this line can be removed.

Reply:
Relates to https://github.com/Homebrew/homebrew-core/pull/44565

Reply:
I've removed it, thx